src/backend/api/Fusion.Resources.Domain/Queries/Departments/GetDepartmentPersonnel.cs

```diff
@@ -141,7 +141,7 @@ public async Task<IEnumerable<QueryInternalPersonnelPerson>> Handle(GetDepartmen
                     .ExpandPositions()
                     .ExpandPositionInstances()
                     .ForAll()
-                    .WithExcludeCompleted()
+                    .WithExcludeCompleted(true)
```

Comment:
default should be true... or...

Reply:
Changed default value to true so method makes more sense